k8s/src/main/scala/io/buoyant/k8s/istio/package.scala

```diff
@@ -1,9 +1,9 @@
 package io.buoyant.k8s
 
 package object istio {
-  val DefaultDiscoveryHost = "istio-manager"
+  val DefaultDiscoveryHost = "istio-pilot"
```

Comment:
The 0.1.6 I downloaded has it as istio-pilot.
I had the exact opposite fear: that someone would deploy istio+linkerd in a namespace other than `default` and be confused why it doesn't work.